### DISCUSSION TITLE
403 Errors on localhost SQL Db

### DISCUSSION THREAD

**Discussion by BumpaRoy** [Azure/data-api-builder#1683] [Q&A] [answered]
I am getting a 403 error when doing a simple Get on a local SQL database.  I was able to get data with the Authors\Books example and another Database on this same server (localhost), but on another database I am receiving 403 errors when trying to do a simple Get on one Entity (https://localhost:5001/api/Product).  The SQL connection string is setup using the "sa" password (the same in all of the Db's I have tried to access).   I am using Postman to initiate the Get.

I honestly don't understand the "anomymous" role and how that ties in with SQL Permissions and why I can access some Db's but not other Db's on the same server with the connection as "sa".  Thank you.

**Comment by seantleonard** [comment]
Hi @bumparoy, the permissions defined in DAB's runtime config do not correlate to the permissions in the connected database, in your case SQL Server.

Can you provide the configuration you are using? Is the anonymous role defined for the entity representing the path `https://localhost:5001/api/Product`?

**Reply by BumpaRoy** [reply, marked as answer]
I do think it had something to do with anonymous, maybe the formatting??  Anyway put it away for the night and got back at it today and it is working.  Not sure what I tried today different that yesterday ..  Just a clearer head I guess.  Thank you for the feedback!

**Reply by seantleonard** [reply]
Glad you got it working! Feel free to open another thread if you encounter the issue again.
